acct-user/zookeeper/zookeeper-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="User for sys-cluster/zookeeper-bin"
+
+ACCT_USER_ID=217
+ACCT_USER_GROUPS=( zookeeper )
```

Comment:
This uid is already taken. Please will a PR at https://github.com/gentoo/api-gentoo-org/tree/master/files. Use `used_free_uidgids.sh` to get a free uid/gid combo.